forward-message-mailgun/functions/forward-message-mailgun.js

```diff
@@ -0,0 +1,33 @@
+const mailgun = require("mailgun-js");
+
+exports.handler = function (context, event, callback) {
+  // Configure these at https://www.twilio.com/console/functions/configure
+  const API_KEY = context.MAILGUN_API_KEY;
+  const DOMAIN = context.MG_VERIFIED_DOMAIN;
+  const TO = context.TO_EMAIL_ADDRESS;
+  const FROM = context.FROM_EMAIL_ADDRESS;
```

Comment:
Since we have "To" and "From" variables from both the `context` and the `event` it might be clearer if we kept these as their original names, `TO_EMAIL_ADDRESS` and `FROM_EMAIL_ADDRESS`.

```javascript
const { TO_EMAIL_ADDRESS, FROM_EMAIL_ADDRESS } = context;
```

And then below, when building up the data, you can use them like this:

```javascript
  const data = {
    from: FROM_EMAIL_ADDRESS,
    to: TO_EMAIL_ADDRESS,
    subject: `SMS received at Twilio number ${To}`,
    text: `SMS received from ${From}. \nMessage: ${Body}`,
}
```